README.md

```diff
@@ -14,7 +14,7 @@
 - [Release Notes](#release-notes)
 
 # Overview
-JFrog CLI is a compact and smart client that provides a simple interface that automates access to *Artifactory*, *Bintray* and *Mission Control* through their respective REST APIs.
+JFrog CLI is a compact and smart client that provides a simple interface that automates access to *Artifactory* and *Mission Control* through their respective REST APIs.
```

Comment:
...a simple interface that automates access to *Artifactory* and *Mission Control* through...
-->
...a simple interface that automates access to *JFrog Artifactory*, *Xray*, *Distribution* and *Mission Control* through...